portality/app.py

```diff
@@ -342,6 +344,15 @@ def page_not_found(e):
 def page_not_found(e):
     return render_template('500.html'), 500
 
+@app.route('/gettoken/<g_recaptcha_response>')
+def verify_recaptcha(g_recaptcha_response):
+    with urllib.request.urlopen('https://www.google.com/recaptcha/api/siteverify?secret=' + app.config.get("RECAPTCHA_SECRET_KEY", "") + '&response=' + g_recaptcha_response) as url:
+        data = json.loads(url.read().decode())
+        return data
+
+@app.route('/get_site_key')
+def get_site_key():
+    return app.config.get('RECAPTCHA_SITE_KEY')
```

Comment:
Is there a better place we can define these routes?

My feeling is we could either introduce a `portality.view.recaptcha` view, giving us a clean place to put them, or maybe they would go in `portality.view.doajservices`. It feels weird having them in the main `app.py`.  There is a bunch of other stuff in app.py, but it's all error handlers, authentication or legacy routes, and I think it would be best to keep it that way.

Reply:
Also, specifically on RECAPTCHA_SITE_KEY, could we add a placeholder in settings.py for that?

When I went to have a look, I noticed we also have RECAPTCHA_PUBLIC_KEY in settings.py, and it looks like that's not used anywhere so we should probably clear it out.

Reply:
> My feeling is we could either introduce a `portality.view.recaptcha` view, giving us a clean place to put them, or maybe they would go in `portality.view.doajservices`. It feels weird having them in the main `app.py`

I've added them now in doaj.py right under contact endpoint implementation. It's only now one method:
```
@app.route('/get_site_key')
def get_site_key():
    return app.config.get('RECAPTCHA_SITE_KEY')
```
used only by contact form. I think for clarity sake it's best to keep them together. Creating separate `portality.view.recaptcha` just for that one short method seems to be overreaction while `portality.view.doajservices` sounds like a good place but at the same time separates the functionality of one small module. What do you think?